config/paths.js

```diff
@@ -4,6 +4,7 @@ const glob = require('fast-glob');
 const { replaceExtension } = require('ds-toolbox-test/tasks/utils');
 
 const CSS_OUTPUT_DIR = './static/build/';
+const SVG_LIB_DIR = './node_modules/ds-toolbox-test/assets/icons/base/';
```

Comment:
No need to update here, we might be better off constructing URLs with `path.join()/resolve()` going forward. I think they're more bulletproof in case files move.